wormhole-connect/src/config/index.ts

```diff
@@ -102,22 +102,22 @@ export function buildConfig(
     ),
     graphql: Object.assign({}, networkData.graphql, customConfig?.graphql),
     wormholeApi: {
-      mainnet: 'https://api.wormholescan.io/',
-      testnet: 'https://api.testnet.wormholescan.io/',
-      devnet: '',
+      Mainnet: 'https://api.wormholescan.io/',
```

Comment:
I just realized these have recently changed to first letter being capital.

Reply:
yes. we switched to the new SDK's type for this recently. :+1: